serviceerror/convert.go

```diff
@@ -94,7 +94,7 @@ func FromStatus(st *status.Status) error {
 		case *failure.MultiOperationExecutionAborted:
 			return newMultiOperationAborted(st)
 		default:
-			// fall through to st.Err()
+			return newAborted(st)
```

Comment:
I am (only slightly) concerned this may be technically incompatible behavior changing an error type that users may be using. Can we make sure that this is _clearly_ called out in the release notes of the next release of this repo?

Reply:
Right; I also thought about that for a bit.

From what I understand, the purpose of `FromStatus` is to convert `status.Status` to a particular type of service error; or fallback to `status.Error` if there isn't one available.

An incompatibility issue would occur if a user 
(1) relied on the error message details (`.Error()`) or
(2) tried to cast the error to a `status.Error`

Both seem unlikely, but definitely technically possible.

👍 I can add a warning to the release notes.